plugins/wme-sitebuilder/wme-sitebuilder/Plugin.php

```diff
@@ -133,4 +136,28 @@ public function registerModule( $module ) {
 
 		return $this;
 	}
+
+	/**
+	 * Add the new admin color scheme.
+	 */
+	public function add_admin_color_scheme() {
```

Comment:
```suggestion
	protected function add_admin_color_scheme() {
```
If this is only being called from this class, it shouldn't be a public method.